pkg/virt-controller/services/template.go

```diff
@@ -178,6 +179,46 @@ func IsCPUNodeDiscoveryEnabled(store cache.Store) bool {
 	return false
 }
 
+func isHyperVSupportRequired(store cache.Store) bool {
```

Comment:
I see the same pattern in the `IsCPUNodeDiscoveryEnabled()` func above, but I'm  wondering why we don't use `pkg/virt-config/feature-gates.go` for this. You already added a new func there, which is unused by now.

Reply:
I like your approach about moving `IsCPUNodeDiscoveryEnabled` into `pkg/virt-config`, but I wonder if it is OK to add a dep to `go-client` (for `cache.Store`) there. If it is OK I'll happily move the related code here. I'm going to kill the unused function, it is just confusing.

Reply:
No need to add go-client there, your function as it is over there already should just work. Have a look at `virt-config/config-map.go`, there the ConfigMap is read, and it's values are put into the env, which are then read in `feature-gates.go`

Reply:
@slintes ack, on it.